climada/hazard/test/test_tc_tracks.py

```diff
@@ -308,6 +308,26 @@ def test_read_legacy_netcdf(self):
             self.assertEqual(tr.basin.shape, tr.time.shape)
             np.testing.assert_array_equal(tr.basin, "SP")
 
+    def test_hdf_io(self):
+        """Test writting and reading hdf5 TCTracks instances"""
+        path = DATA_DIR.joinpath("tc_tracks.h5")
+        tc_track = tc.TCTracks.from_ibtracs_netcdf(
+            provider='usa', year_range=(1993, 1994), basin='EP', estimate_missing=True)
+        tc_track.write_hdf(path)
```

Comment:
This is only testing for the ibtracs. But, seeing that the methods for gettelman and emanuel were also changed, can one do a test for these too? Or is there an issue with the available datasets?

Reply:
The change only affects the dtype of the `basin` variable. But yes, let's add unit tests for that.